build/grafana/dashboard-gameservers.yaml

```diff
@@ -1114,7 +1114,7 @@ data:
       },
       "timezone": "",
       "title": "Agones GameServers",
-      "uid": "D78OotUmk",
+      "uid": "D78OotUmkk",
```

Comment:
Dumb question since I'm not familiar with Grafana, why `uid` matters for reimporting?

Reply:
I tested it again on main branch with the original UID and it works when I apply the new configmap that contains the new graph for this GameServer dashboard. At the beginning I thought there was a conflict with the UID but I think I misunderstood how it works because i was actually importing the JSON in Grafana and that's where it was complaining about the UID. being the same even tho the configmap wasn't.